### PR TITLE
Create on boarding screen & animations, Format translation json file

### PR DESCRIPTION
**Pull request by chuwong35122**
**Changes**
- create onboarding screen page (collab with Gun)
- - Change format of i18n translation JSON.
`
{
  "home": {
    "userGreet": "Hello,",
    "settingButton": "Settings"
  },
}
`
- To call the translation
`
console.log(t('home.userGreet'));
`

**IMPORTANT NOTE**
- Note: There is no implementation of user instance yet. Please refer to the bottom part to navigate to home manually.
- To navigate to home > In MainNavigation.tsx, set user to true. React navigation will automatically render the Home screen.

`const [user, setUser] = useState(null);

  {user ? (
          <>
          ...
          </>
        ) : (
          <>
            <Stack.Screen
              name="OnBoardingScreen"
              component={OnBoardingScreen}
              options={{ headerShown: false }}
            />
          </>
        )}`

### CODE REVIEW

File: src/hooks/useSettings.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const useSettings = () => {
   const [isSoundEffectOn, setIsSoundEffectOn] = useState(true);
 
   const changeLanguage = (lang: string) => {
+    console.log(lang);
```
Comment (pointmekin) [inline src/hooks/useSettings.tsx]: remove

File: src/internationalization/useLanguage.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ export const useLanguage = () => {
   };
 
   const changeLanguage = (lang: string) => {
+    console.log(lang);
```
Comment (pointmekin) [inline src/internationalization/useLanguage.tsx]: Remove

File: src/components/atoms/Input.tsx [hunk 3]
```diff
@@ -9,12 +9,27 @@ const InputBox = ({ name }: InputBoxProps) => {
...
+      <Box alignItems="center" h={16} mb={2}>
+        <Input
+          borderColor="#ACB5BD"
+          placeholder={name}
+          w="100%"
+          size="xl"
+          fontSize="sm"
+          padding={3}
+        />
+        {/* When form validation is implemented, use this text style */}
```
Comment (pointmekin) [inline src/components/atoms/Input.tsx]: Invalid now. Pls remove.

**Review by pointmekin — CHANGES_REQUESTED** [review]

**Review by SPK137 — APPROVED** [review]
Looks okay na gogo

**Review by pointmekin — APPROVED** [review]
Let's go

**Pull request merged**